README.md

```diff
@@ -183,6 +190,24 @@ You can also install tyssue from PyPi, this is a CGAL-less version (pure python)
 
 See [INSTALL.md](INSTALL.md) for a step by step install, including the necessary python environment.
 
+## How to cite 
+If you find `tyssue` useful please cite [this repository](https://github.com/DamCB/tyssue) using its DOI as follows:
+
+> Theis, Suzanne, Gay, (2021). Tyssue: an epithelium simulation library. Journal of Open Source Software, 6(62), 2973 doi:[https://doi.org/10.21105/joss.02973](https://doi.org/10.21105/joss.02973)
+> 
+> Zenodo doi:10.5281/zenodo.4817609
+
+## Publications
+If `tyssue` has made a substantial contribution to your work, please let us know about your publication, and we’ll add your article here.
```

Comment:
"Please edit this file and open a PR"

Also this was published using tyssue:

https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4052248

Reply:
I use reference to tyssue.bib but the display is not working. Do you know if I miss something to use it correctly ? 
If it is just not working. I think I will remove references to tyssue.bib because it is not very nice in the text as it is now.

Reply:
yeah I am not updating that bib file much, we can just remove it